src/CMakeLists.txt

```
@@ -3,8 +3,8 @@ add_subdirectory(cpprealm/internal/bridge/generator)
 include(GetVersion)
 # Configure source code to use right version number
 configure_file(${PROJECT_SOURCE_DIR}/src/cpprealm/internal/version_numbers.hpp.in ${PROJECT_SOURCE_DIR}/src/cpprealm/internal/version_numbers.hpp)
-install(FILES version_numbers.hpp
-        DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/cpprealm
+install(FILES ${PROJECT_SOURCE_DIR}/src/cpprealm/internal/version_numbers.hpp
```

```suggestion
install(FILES cpprealm/internal/version_numbers.hpp
```
The path is relative to the current CMakeLists.txt, it doesn’t need the variable. Avoid using `PROJECT_SOURCE_DIR` because it can have unexpected meanings if this is a subproject. Prefer the variable that has the name given to `project()` like `RealmCxx_SOURCE_DIR`.